### ISSUE TITLE
disconnect in version numbers

### ISSUE THREAD

**Issue by jbathegit** [NOAA-EMC/NCEPLIBS-bufr#27]
The latest tagged release of the software is 11.4.0; however, the VERSION file still says its 11.3.1.1, and the src/bvers.f file still says its 11.3.0.  We need to make sure that all of these are in sync.  Ideally, when incrementing the version number, it would be nice if we could do it in just one place and then have that new number automatically propogate everywhere else.

**Comment by edwardhartnett** [comment]
Also see #21 

Seems like the way to go is to have the VERSION file be the canonical source for the version number. That's where the build system and the documentation get it. I have entered a new issue for that change: #28.

Once that is done, the bvers.f file cannot be out of sync.

We still have to take care when releasing a new version that the VERSION file correctly matches the tag (and release name) we use in GitHub.

**Comment by jbathegit** [comment]
fixed within #30, as commit 04039e2
